### PR TITLE
Aug 2021 reference: future to past tense

### PR DESCRIPTION
**Pull request by srsmsft**
"Will happen in" to "should have happened in" for Aug 2021 date

### CODE REVIEW

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ When a meeting is recorded, it's automatically:
 Related: [Teams meeting recording end user documentation](https://support.microsoft.com/en-us/office/record-a-meeting-in-teams-34dfbe7f-b07d-4a27-b4c6-de62f1348c24)
 
 >[!Note]
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:39]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @srsmsft, kindly commit the additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you, @srsmsft.

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
Closing.  This does not meet the bar.  Also, the PR was submitted with branch conflicts.

**Pull request closed without merging**